script.js

```diff
@@ -0,0 +1,47 @@
+// var gameBoard = ["topleft", "topmid", "topright", "midleft", "midmid", "midright", "botleft", "botmid", "botright"];
+
+// gameBoard.addEventListener("click", xOrO, false);
+//
+// function xOrO () {
+//     for(i = 0; i < gameBoard.length; i++) {
+//       if (i === (i % 2 === 1)) {
+//         return ("X");
+//       }
+//       else {
+//         return ("O")
+//       }
+// }
+//got hint to create a turn counter instead to track turn value
+
+//comment out everthing and start over
+
+var board = document.getElementsByClassName("gameBoard");
+var turn = 0;
+
+//While there are empty squares on the board, make them clickable.  When one is clicked, call the xOrO function.
+for (var i = 0; i < board.length; i++) {
+  var squares = board.children[i];
```

Comment:
At this point, `board` is an array of each element with the class name "gameBoard". An array doesn't have a `children` property. What you wanted was `board[i]`. That's a little confusing, so I would have rewritten it like this:

```
var squares = document.getElementsByClassName("gameBoard");
for (var s = 0; s < squares.length; s++){
  var square = squares[s];
  // and so on
}
```